.github/workflows/build-intermediary-image.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and push intermediary Docker image
+
+on:
+  pull_request:
```

Comment:
are we creating everytime on PRs?

Reply:
Nope it is done like that only for testing purposes, I'm going to remove it eventually and only leave the `workflow_dispatch` trigger.

Reply:
Fixed, the `pull_request` (and `push`) triggers are removed, as it was solely for testing purposes.